msdsl/model.py

```diff
@@ -475,15 +477,19 @@ def set_from_sync_func(self, signal: Union[Signal, str], func: GeneralFunction,
 
     # table creation functions
 
-    def make_real_table(self, vals, width=18, exp=None, name=None, dir=None):
+    def make_real_table(self, vals, width=18, exp=None, name=None, dir=None,
```

Comment:
there is usually no need to define **real_type**, since it defaults to the value from the class instantiation